public/app/themes/clarity/src/globals/js/auth-heartbeat.js

```diff
@@ -0,0 +1,6 @@
+export default (function ($) {
+    // Sent a request to the heartbeat endpoint, this will refresh the oauth token.
+    setInterval(function(){
+        $.get( "/auth/heartbeat" )
```

Comment:
If 401, modal - your session has ended, please login again for security purposes. Blur bg. Redirect in 5s. 

The js can just reload the current page.

Change to 10s